### PR TITLE
fix(css): add layer() function page

### LINKED ISSUE #35005: Add layer() function page and define `layer` keyterm.
We need to add a `layer()` function page, and add `layer` keyterm to the `@import` page where we describe values.

```suggestion
use {{cssxref("@import")}} together with the `layer` keyword or `layer()` function to import external style sheets (from frameworks, widget stylesheets, libraries, etc.) into layers. 
```

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- fix https://github.com/mdn/content/issues/35005

### CODE REVIEW

File: files/en-us/web/css/@import/layer_function/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: layer()
+slug: Web/CSS/@import/layer_function
+page-type: css-function
+browser-compat: css.at-rules.import.layer
+---
+
+{{CSSRef}}
+
+The **`layer()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS_Functions) is used along with the [`@import`](/en-US/docs/Web/CSS/@import) at-rule to put the imported resource in a separate named [cascade layer](/en-US/docs/Web/CSS/@layer).
```
Comment (bsmth) [inline files/en-us/web/css/@import/layer_function/index.md:10]: I think we're also linking "at-rule" in other docs

File: files/en-us/web/css/@import/index.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ where:
     If the browser does not conform to the conditions specified in the _supports-condition_, it may not fetch the linked stylesheet, and even if downloaded through some other path, will not load it.
     The syntax of `supports()` is almost identical to that described in {{CSSxRef("@supports")}}, and that topic can be used as a more complete reference.
 
+Use {{cssxref("@import")}} together with the `layer` keyword or `layer()` function to import external style sheets (from frameworks, widget stylesheets, libraries, etc.) into layers.
```
Comment (bsmth) [inline files/en-us/web/css/@import/index.md:40]: this is a self-link for `@import`

File: files/en-us/web/css/@import/layer_function/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/@import/layer_function/index.md:10]: > I think we're also linking "at-rule" in other docs

```suggestion
The **`layer()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS_Functions) is used along with the [`@import`](/en-US/docs/Web/CSS/@import) [at-rule](/en-US/docs/Web/CSS/At-rule) to put the imported resource in a separate named [cascade layer](/en-US/docs/Web/CSS/@layer).
```

File: files/en-us/web/css/@import/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/@import/index.md:40]:
```suggestion
Use `@import` together with the `layer` keyword or `layer()` function to import external style sheets (from frameworks, widget stylesheets, libraries, etc.) into layers.
```

**Review by bsmth — APPROVED** [review]
We're missing examples, but otherwise +1 from me, ty 👍🏻

**Pull request merged**